src/DynamoCore/Core/CustomNodeManager.cs

```diff
@@ -398,7 +398,20 @@ public IEnumerable<CustomNodeInfo> AddUninitializedCustomNodesInPath(string path
             {
                 info.IsPackageMember = isPackageMember;
 
-                SetNodeInfo(info, isTestMode);
+                SetNodeInfo(info);
+                result.Add(info);
+            }
+            return result;
+        }
+
+        public IEnumerable<CustomNodeInfo> AddUninitializedCustomNodesInPath(string path, bool isTestMode, PackageInfo packageInfo)
```

Comment:
Would you add some comments for this new API?

Reply:
will do, thanks for catching.